web/modules/custom/mymodule/mymodule.libraries.yml

```diff
@@ -0,0 +1,5 @@
+myplugin:
+    version: 1.x
+    css:
+        theme:
+            css/myplugin.css: {}
```

Comment:
@anitha-innoraft - This doesn't follow the YAML code standards. There should only be a 2 space indentation. Please fix.

Reply:
Fixed.